pom.xml

```diff
@@ -4389,6 +4389,14 @@
                <artifactId>os-maven-plugin</artifactId>
                <version>1.4.1.Final</version>
             </plugin>
+            <plugin>
+               <groupId>org.infinispan.maven-plugins</groupId>
+               <artifactId>proto-schema-compatibility</artifactId>
+               <version>${version.infinispan.maven-plugins}</version>
+               <configuration>
+                  <commitProtoLock>${commitProtoLockChanges}</commitProtoLock>
```

Comment:
Please define the property as false in the POM, otherwise IntelliJ flags it as an error